usaspending_api/search/tests/data/utilities.py

```diff
@@ -5,6 +5,9 @@ def setup_elasticsearch_test(monkeypatch, index_fixture, **options):
     if index_fixture.index_type == "awards":
         search_wrapper = "AwardSearch"
         query_alias = settings.ES_AWARDS_QUERY_ALIAS_PREFIX
+    elif index_fixture.index_type == "covid19_faba_":
+        search_wrapper = "AccountSearch"
+        query_alias = settings.ES_COVID19_FABA_QUERY_ALIAS_PREFIX
     else:
```

Comment:
Similar to a previous comment - now that we're expanding our es ETL code to handle arbitrary ES data types, we shouldn't have conditional statements ending on `else: ...`. Should be more like a case statement: either it matches one or fails.